ray/requirements.txt

```diff
@@ -44,7 +44,8 @@ tensorboard==2.11.2
 tensorboard-data-server==0.6.1
 tensorboard-plugin-wit==1.8.1
 tensorflow-estimator==2.11.0
-tensorflow==2.11.0
```

Comment:
https://stackoverflow.com/questions/71688065/generic-requirements-txt-for-tensorflow-on-both-apple-m1-and-other-devices